less/index.less

```diff
@@ -49,11 +49,262 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+
+@mainColor: #F2F2F2;
+@accentColor:  #69e0c0;
+.flex-layout(@flex, @padding){
```

Comment:
cool idea for a mixin, but I'm not sure why you're passing a flex parameter? If you want it to set to flex, don't have a flex parameter, and just set it to display: flex in the style